src/shell/shell_misc.cpp

```diff
@@ -50,6 +51,15 @@ static void outc(Bit8u c) {
 	DOS_WriteFile(STDOUT,&c,&n);
 }
 
+static void backone() {
+	BIOS_NCOLS;
+	uint8_t page = real_readb(BIOSMEM_SEG, BIOSMEM_CURRENT_PAGE);
```

Comment:
(minor) if we're not changing `page` later, suggest making it read-only `const`.

Reply:
Done.